sys/dev/athn/athn.c

```diff
@@ -735,6 +740,7 @@ athn_reset(struct athn_softc *sc, int cold)
 	}
```

Comment:
Also on line 738 there should be ATHN_UNLOCK()

Reply:
Done, thx!